web-client/src/views/FileDocument/NonstandardForm.tsx

```diff
@@ -189,136 +260,8 @@ export const NonstandardForm = connect(
         )}
         {helper[level].ordinalField && (
           <>
-            <NonMobile>
-              <FormGroup className="grid-row grid-gap">
```

Comment:
This FormGroup--specifically the grid-row class--was the issue. I also refactored to avoid more or less duplicating 30-some lines of code.